packages/admin_view/src/components/LexiconTable.tsx

```diff
@@ -0,0 +1,118 @@
+import { GetAllLexEntriesQuery, useGetAllLexEntriesQuery } from '../graphql/lexicon/lexicon.ts';
+import { DataGrid, GridColDef,DataGridProps } from '@mui/x-data-grid';
+import React, { useState } from 'react';
+import { Box, Stack, Button } from '@mui/material';
+import { alpha, createTheme, styled } from '@mui/material/styles';
+
+const theme = createTheme({
+  palette:{
+    primary:{
+      light: '#757ce8', 
+      main: '#3f50b5', 
+      dark: '#002884',   
+    }
+  }
+})
+const StyledDataGrid = styled(DataGrid)<DataGridProps>(({ theme }) => ({
+  '&.MuiDataGrid-root': {
```

Comment:
I'd recommend removing the custom CSS. We should be able to get the functionality we want with the Global theme when the time comes